File: Week 6/pset6/cash/cash.py
from cs50 import get_float

while True:
    cash = get_float("Change owed: ") * 100
    if cash >= 0:
        break
    
coins = 0
while (cash > 0):
    if cash < 5:
        cash -= 1
    elif cash < 10:
        cash -= 5
    elif cash < 25:
        cash -= 10
    elif cash >= 25:
        cash -= 25
    coins += 1

print(coins)

File: Week 1/pset1/cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

// cash (1$, 5%, 10$, 25$)

int main(void)
{
    int cash;
    
    do
    {
        cash = round(get_float("Cash: ") * 100);
    } while(cash < 0);
    
    int coins = 0;
    
    while (cash > 0)
    {
        if (cash < 5) {
            cash -= 1;
        } else if (cash < 10) {
            cash -= 5;
        } else if (cash < 25) {
            cash -= 10;
        } else if (cash >= 25) {
            cash -= 25;
        }
        coins++;
    }
    
    printf("%d\n", coins);
    
    return 0;
}